CorsixTH/Lua/config_finder.lua

```diff
@@ -573,6 +573,7 @@ local hotkeys_defaults = {
   ingame_zoom_in_more = {"shift", "="},
   ingame_zoom_out = "-",
   ingame_zoom_out_more = {"shift", "-"},
+  ingame_reset_zoom = "0",
```

Comment:
In regards to the other zoom controls I would think we should have it as ``shift + 0`` . Unless that's already taken.

Reply:
Actually the two different zoom commands confused me. Maybe just 0 is fine.

Reply:
Yeah, "=" and "-" without holding shift zoom in and out, so it made sense to me that 0 wouldn't require shift either.